### PR TITLE
testing: when using signed-client yml, use client container with :pr tag

### PR DESCRIPTION
**Pull request by GregorioDiStefano**
Changelog: none
Signed-off-by: Greg Di Stefano <greg.distefano+github@gmail.com>

### CODE REVIEW

File: extra/signed-artifact-client-testing/docker-compose.signed-client.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ services:
     # mender-client
     #
     mender-client:
-        image: mendersoftware/mender-client-qemu:master
+        image: mendersoftware/mender-client-qemu:pr
```
Comment (kacf) [inline extra/signed-artifact-client-testing/docker-compose.signed-client.yml]: This is added on top of the other docker compose files, right? In that case `image` should not be specified at all, you should just rely on the existing one.

Comment (GregorioDiStefano) [inline extra/signed-artifact-client-testing/docker-compose.signed-client.yml]: didn't know that, thanks.

**Comment by kacf** [comment]
@GregorioDiStefano: Doesn't the token stuff belong in #289?

**Comment by GregorioDiStefano** [comment]
Ooops, fixed now :) Just included a commit by accident.

**Comment by kacf** [comment]
This has been fixed already.

**Pull request closed without merging**